crates/swc_ecma_minifier/src/option/mod.rs

```diff
@@ -60,6 +60,9 @@ pub struct MangleOptions {
 
     #[serde(default, alias = "safari10")]
     pub safari10: bool,
+
+    #[serde(default, alias = "reserved")]
+    pub reserved: Vec<String>,
```

Comment:
I think it's better to use `JsWord` here.

Reply:
You can read about it at https://rustdoc.swc.rs/swc_atoms/

Reply:
Thanks for the link! 

Updated to use `JsWord`. I also updated the existing `properties.reserved` to use it too https://github.com/swc-project/swc/pull/3476/files#diff-b40213014998d084ab9d7c55540e0646fb34d56f60e8d400edf83457c78bb5c8R72